transaction/std/src/input_credentials.rs

```diff
@@ -84,3 +85,17 @@ impl InputCredentials {
         })
     }
 }
+
+impl Zeroize for InputCredentials {
+    fn zeroize(&mut self) {
```

Comment:
Should this zeroize `real_output_public_key` as well? Otherwise, unless I'm mistaken, the `real_index` could be determined.

Reply:
The type backing `RistrettoPublic` (`RistrettoPoint`) does not support zeroizing and is generally not deemed sensitive. I honestly don't think `real_index` is sensitive either but since it was possible to zeroize it I went for it. The idea is to protect keys that, if leaked, could allow someone to steal your money. I don't think privacy is a concern, since this is all running on clients machines where there are ample other ways to degrade privacy.

Reply:
Should we not zeroize `real_index` then? Seems misleading otherwise

Reply:
I'm fine removing that zeroize.